README.md

```diff
@@ -120,9 +120,10 @@ npm run lint
 | REDIS_URL **(required)** | Redis server URL, including port and protocol | |
 | REDIS_HOST **(required)** | Redis hostname. Can be used instead of `REDIS_URL`. Will override `REDIS_URL` if set | |
 | REDIS_AUTH_TOKEN | Optional auth token for the Redis instance | |
-| API_BASE_URL **(required)** | Base URL for the backend API server for this service without any path | `http://localhost:3000/api/v1` |
-| API_HEALTHCHECK_URL **(required)** | URL to which healthcheck pings are sent | |
-| API_AUTH_URL **(required)** | URL to which OAuth2 access token requests should be sent | |
+| API_BASE_URL **(required)** | Base URL for the backend API server for this service without any path | `http://localhost:3000` |
+| API_PATH **(required)** | Base base for the API | `/api/v1` |
```

Comment:
Not part of this work but I wonder if we need to do an `env` check on required on `envs` when the app starts up. Then error if they are not present?

Reply:
Yeah might be worth having. But then we'd need a way to store required envs and maintain that list.